docs/userguide/citing.rst

```diff
@@ -17,28 +17,40 @@ Example BibTeX entry:
 .. parsed-literal::
 
   @software{metpy,
-    author = {May, Ryan M. and Arms, Sean C. and Marsh, Patrick and Bruning, Eric and Leeman, John R.
-              and Goebbert, Kevin and Thielen, Jonathan E. and Bruick, Zachary S.},
+    author = {May, Ryan M. and Arms, Sean C. and Marsh, Patrick and Bruning, Eric and Leeman,
+              John R. and Goebbert, Kevin and Thielen, Jonathan E. and Bruick, Zachary S.},
     organization = {Unidata},
     title = {MetPy: A {Python} Package for Meteorological Data},
-    year = {2008 - |cite_year|},
+    year = {|cite_year|},
     version = {|cite_version|},
     doi = {10.5065/D6WW7G29},
     url = {https://github.com/Unidata/MetPy},
-    address = {Boulder, Colorado}
   }
 
 
 *****************
 AMS Journal Style
 *****************
 
+AMS suggested BibTeX entry:
+
+.. parsed-literal::
+
+  @misc{metpy,
+    author = {May, Ryan M. and Arms, Sean C. and Marsh, Patrick and Bruning, Eric and Leeman,
+              John R. and Goebbert, Kevin and Thielen, Jonathan E. and Bruick, Zachary S.},
+    publisher = {Unidata},
+    title = {MetPy: A {Python} Package for Meteorological Data},
+    year = {|cite_year|},
+    doi = {10.5065/D6WW7G29},
+    url = {https://github.com/Unidata/MetPy},
+  }
+
+
 Example Citation:
 
 .. parsed-literal::
 
   May, R. M., Arms, S. C., Marsh, P., Bruning, E., Leeman, J. R., Goebbert, K., Thielen, J. E.,
       and Bruick, Z., |cite_year|: MetPy: A Python Package for Meteorological Data.
-      Version |cite_version|, Unidata, Accessed |access_date|.
-      [Available online at https://github.com/Unidata/MetPy.]
-      doi:10.5065/D6WW7G29.
+      Unidata, URL https://github.com/Unidata/MetPy, doi:10.5065/D6WW7G29.
```

Comment:
so that is yields,

`Unidata, accessed 18 December 2020, https://github.com/Unidata/MetPy, doi:10.5065/D6WW7G29.`

Reply:
Current AMS guidance actually also only references the DOI web address, but I still like putting the actual address in for this since it isn't likely to change and would always be able to be found through the DOI number.

Reply:
I'm good with going the `NOTE=...` route. Just need to resolve the access question above.